media/css/firefox/windows-64-bit.scss

```diff
@@ -0,0 +1,76 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/article';
+@import '../../protocol/css/components/call-out';
+
+h1, h2 {
+    @include open-sans;
+    @include text-display-lg;
+}
+
+.mzp-c-hero {
+    background: #33124e;
+    color: #fff;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
The title could also maybe use a bit more breathing room on mobile? It looks ok in English, but if this page we're translated it might be a little too tight as it stands?

![mobile](https://user-images.githubusercontent.com/400117/53175225-6f242980-35e3-11e9-9a60-b4aa8954c15f.png)